aocd/models.py

```diff
@@ -54,7 +54,7 @@ def auth(self):
 
     @property
     def id(self):
-        fname = os.path.join(AOCD_DIR, "token2id.json")
+        fname = os.path.join(AOCD_DATA_DIR, "token2id.json")
```

Comment:
Wrong dir - this one is config, not data.

Reply:
Is token2id config? given it isn't created or edited by the user?

Reply:
The thing that I think is adding some value in this pull request is the separation of _cache_ vs _auth_. While you're right that `token2id.json` is an auto-generated file, and is in some sense still just a cache (one user id -> many tokens) this file does have secrets in it.

A user might reasonably want any file with their config/tokens to be mode 0o600 (private) but the cache data such as puzzle inputs/answers/titles to be mode 0o644 (readable).